.readthedocs.yaml

```diff
@@ -9,7 +9,7 @@ sphinx:
   configuration: docs/conf.py
 
 build:
-  os: "ubuntu-22.04"
+  os: "ubuntu-24.04"
```

Comment:
Don't do this.

ReadTheDocs uses completely different CI than GitHub Actions. We need to see the supported OS version of ReadTheDocs.